Kernel/KLexicalPath.cpp

```diff
@@ -8,6 +8,8 @@
 
 namespace Kernel::KLexicalPath {
 
+static char const* const s_single_dot = ".";
```

Comment:
Can this be a `StringView` literal instead?

Reply:
Yes it can, you're right.